src/core/reference/include/openvino/reference/convert.hpp

```diff
@@ -70,6 +71,11 @@ TO get_value(const uint8_t* buf, size_t idx, element::Type from_type) {
     if (from_type == element::i4) {
         return detail::get_i4(buf, idx);
     }
+
+    if (from_type == element::nf4) {
+        return detail::get_u4(buf, idx);
```

Comment:
do we need get_nf4 ?

Reply:
Removed this code for changing the order of elements in packing.